weave-js/src/components/PagePanelComponents/Home/Home.tsx

```diff
@@ -221,7 +221,8 @@ const HomeComp: FC<HomeProps> = props => {
     ...REDIRECT_WANDB,
     ...REDIRECT_LOCAL,
   ];
-  const {pathname} = window.location;
+  let {pathname} = window.location;
+  pathname = pathname.substring(window.WEAVE_CONFIG.PREFIX.length);
```

Comment:
We should import getConfig from `config.ts` that will guarantee `PREFIX` is available, i.e. `getConfig().PREFIX`